remote/setup_workspace.py

```diff
@@ -22,7 +22,9 @@
 )
 
 
-def replace_config(path: pathlib.Path, key: str, value: str) -> None:
+def replace_config(
+    logger: logging.Logger, path: pathlib.Path, key: str, value: str
```

Comment:
When `LOGGER` is global, you can use it directly. No need to pass it to the method.

Reply:
This comes from when I tried to isolate the function in its own file for reusability. Fixed.